src/jenkins_job_insight/main.py

```
@@ -4776,6 +4797,90 @@ async def analyze_comment_intent(
+        )
+    try:
+        ai_provider, ai_model = _resolve_ai_config_values(None, None)
+    except HTTPException:
+        raise HTTPException(
+            status_code=503,
+            detail=(
+                "AI provider not configured on this server. "
+                "Configure AI_PROVIDER and AI_MODEL environment variables "
+                "to enable AI-powered feedback."
+            ),
+        )
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)